blocks/components/editable/index.js

```diff
@@ -1,3 +1,75 @@
-export default function Editable( { value } ) {
-	return <p>{ value }</p>;
+export default class Editable extends wp.element.Component {
+	constructor() {
+		super( ...arguments );
+		this.onInit = this.onInit.bind( this );
+		this.onSetup = this.onSetup.bind( this );
+		this.onChange = this.onChange.bind( this );
+		this.bindNode = this.bindNode.bind( this );
+	}
+
+	componentDidMount() {
+		this.initialize();
+	}
+
+	initialize() {
+		const config = {
+			target: this.node,
+			theme: false,
+			inline: true,
+			toolbar: false,
+			browser_spellcheck: true,
+			entity_encoding: 'raw',
+			setup: this.onSetup,
+			formats: {
+				strikethrough: { inline: 'del' }
+			}
+		};
+
+		tinymce.init( config );
+	}
+
+	onSetup( editor ) {
+		this.editor = editor;
+		editor.on( 'init', this.onInit );
+		editor.on( 'focusout', this.onChange );
```

Comment:
Since this is a "controlled" TinyMCE (each time the content prop changes, we update the content), I had issues while using `change` on the `per-block` prototype. I haven't tried `blur` though